install/openshift-web-console-operator/install.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+parameters:
+- name: IMAGE
+  value: openshift/origin:latest
+- name: NAMESPACE
+  # This namespace must not be changed.
+  value: openshift-core-operators
+- name: LOGLEVEL
+  value: "0"
+- name: COMPONENT_LOGLEVEL
+  value: "0"
+- name: COMPONENT_IMAGE
+  value: openshift/origin-web-console:latest
+- name: NODE_SELECTOR
+  value: "{}"
+objects:
+
+- apiVersion: apiextensions.k8s.io/v1beta1
+  kind: CustomResourceDefinition
+  metadata:
+    name: openshiftwebconsoleconfigs.webconsole.operator.openshift.io
+  spec:
+    scope: Cluster
+    group: webconsole.operator.openshift.io
+    version: v1alpha1
+    names:
+      kind: OpenShiftWebConsoleConfig
+      plural: openshiftwebconsoleconfigs
+      singular: openshiftwebconsoleconfig
+
+- apiVersion: apps/v1beta1
+  kind: Deployment
+  metadata:
+    namespace: ${NAMESPACE}
+    name: openshift-web-console-operator
+    labels:
+      app: openshift-web-console-operator
+  spec:
+    replicas: 1
+    template:
+      metadata:
+        name: openshift-web-console-operator
+        labels:
+          app: openshift-web-console-operator
+      spec:
+        serviceAccountName: openshift-web-console-operator
+        containers:
+        - name: operator
+          image: ${IMAGE}
+          imagePullPolicy: IfNotPresent
+          command: ["hypershift", "experimental", "openshift-webconsole-operator"]
+          args:
+          - "-v=${LOGLEVEL}"
+        nodeSelector: "${{NODE_SELECTOR}}"
+
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    namespace: ${NAMESPACE}
+    name: openshift-web-console-operator
+    labels:
+      app: openshift-web-console-operator
+
+- apiVersion: webconsole.operator.openshift.io/v1alpha1
+  kind: OpenShiftWebConsoleConfig
+  metadata:
+    name: instance
+  spec:
+    managementState: Managed
+    imagePullSpec: ${COMPONENT_IMAGE}
+    version: 3.10.0
```

Comment:
@deads2k will this need to be bump every release or we can somehow pass the version as parameter?

Reply:
> @deads2k will this need to be bump every release or we can somehow pass the version as parameter?

I'm actually not sure what to do yet.  This is demonstrating out problem.